web/src/message_list_view.ts

```diff
@@ -1696,13 +1696,21 @@ export class MessageListView {
         // call; it was introduced in an earlier version of this code
         // where we constructed an artificial message group for this
         // rerendering rather than looking up the original version.
```

Comment:
I'd prefer to follow this `TODO` and remove this block entirely, but would like some other input on that before doing it. Asked more about that here: https://chat.zulip.org/#narrow/stream/9-issues/topic/Blueslip.20error.20on.20editing.20a.20message